debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlGeometry.java

```diff
@@ -5,28 +5,34 @@
  */
 package io.debezium.connector.mysql;
 
-import java.util.Arrays;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import mil.nga.wkb.geom.Point;
-import mil.nga.wkb.io.ByteReader;
-import mil.nga.wkb.io.WkbGeometryReader;
 
 /**
- * A parser API for MySQL Geometry types, it uses geopackage-wkb-java as a base for parsing Well-Known Binary
+ * A parser API for MySQL Geometry types
  *
  * @author Omar Al-Safi
+ * @author Robert Coup
  */
 public class MySqlGeometry {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(MySqlGeometry.class);
 
     private final byte[] wkb;
+    private final int srid;
```

Comment:
Could you add JavaDoc here? The geo types are not natural to everyone (not to me for sure ;), so that will help.

Reply:
Willdo. SRID refers to a ["Spatial Reference System Identifier"](https://en.wikipedia.org/wiki/Spatial_reference_system) — basically which coordinate system a particular geometry representation is using. There's a big list of numbers ("EPSG codes") maintained by the International Association of Oil and Gas Producers which are the standard ones, but at the end of the day it's an arbitrary integer.